internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/server/TestBackupControllerResponse.java

```diff
@@ -27,6 +30,8 @@ public class TestBackupControllerResponse {
   @Test
   public void backupControllerThrows421() throws Exception {
     String clusterName = "backupControllerThrows421";
+    Properties extraProperties = new Properties();
+    extraProperties.put(ConfigKeys.CONTROLLER_PARENT_REGION_STATE, ParentControllerRegionState.ACTIVE.name());
```

Comment:
We shouldn't need this for child controllers. Also, we shouldn't need to update so many tests since we made the default value as `ACTIVE`, right?

Reply:
This is correct. Removed from pr and test passes without it.